base/docs/basedocs.jl

```diff
@@ -1637,9 +1637,36 @@ julia> ex = ErrorException("I've done a bad thing");
 julia> ex.msg
 "I've done a bad thing"
 ```
+
+!!! compat "Julia 1.12"
+    ErrorException should no longer be used to raise exceptions for invalid field access. Use FieldError instead.
+
 """
 ErrorException
 
+"""
+    FieldError(type, field)
```

Comment:
```suggestion
    FieldError(type::DataType, field_name::Symbol)
```

Reply:
Do you mind If i keep `field` name as is. Original FieldError has `field` as field name. It will be consistent if we keep it unchanged.

Reply:
Sure, fine. Just making suggestions. My main point is that it's nice to see the argument types in the docs.